tcs/csp_solver_core.h

```diff
@@ -686,20 +671,25 @@ class C_csp_tes
 	struct S_csp_tes_outputs
 	{
 		double m_q_heater;			//[MWe]  Heating power required to keep tanks at a minimum temperature
-        double m_m_dot;             //[kg/s] Hot tank mass flow rate, valid for direct and indirect systems
-		double m_W_dot_rhtf_pump;	//[MWe]  Pumping power, just for tank-to-tank in indirect storage
-		double m_q_dot_loss;		//[MWt]  Storage thermal losses
 		double m_q_dot_dc_to_htf;	//[MWt]  Thermal power to the HTF from storage
 		double m_q_dot_ch_from_htf;	//[MWt]  Thermal power from the HTF to storage
-		double m_T_hot_ave;		    //[K]    Average hot tank temperature over timestep
-		double m_T_cold_ave;	    //[K]    Average cold tank temperature over timestep
-		double m_T_hot_final;	    //[K]    Hot tank temperature at end of timestep
-		double m_T_cold_final;	    //[K]    Cold tank temperature at end of timestep
-	
+		
+		double m_m_dot_cr_to_tes_hot;	//[kg/s]
+		double m_m_dot_tes_hot_out;	//[kg/s]
+		double m_m_dot_pc_to_tes_cold;	//[kg/s]
+		double m_m_dot_tes_cold_out;		//[kg/s]
+		double m_m_dot_field_to_cycle;	//[kg/s]
+		double m_m_dot_cycle_to_field;	//[kg/s]
+
+		// Mass flow rate from one tank directly to another. = 0 for direct systems
+		double m_m_dot_cold_tank_to_hot_tank;	//[kg/s] 
```

Comment:
Time to start removing the m_ prefix?  The right-click->Rename works well.

Reply:
it is not time